drv/monorail-api/src/lib.rs

```diff
@@ -130,6 +130,9 @@ pub enum MonorailError {
     /// The given port does not have a PHY associated with it
     NoPhy,
 
+    /// The given function is not implemented
+    NotImplemented,
```

Comment:
Under what circumstances would a correct program call an unimplemented function in monorail? (i.e. should this be replaced by reply-fault?)

Reply:
Switched to reply-fault in 45a93d961a14